sdk/eventgrid/azure-messaging-eventgrid/swagger/README.md

```diff
@@ -495,5 +574,65 @@ public class EventGridCustomization extends Customization {
         }
         return result.toUpperCase();
     }
+
+    public void customizeEventGridClientImplImports(LibraryCustomization customization) {
+        PackageCustomization packageModels = customization.getPackage("com.azure.messaging.eventgrid.implementation");
+        ClassCustomization classCustomization = packageModels.getClass("EventGridPublisherClientImpl");
+    
+        classCustomization.customizeAst(comp -> {
+            comp.getImports().removeIf(p -> {
+                return p.getNameAsString().equals("com.azure.messaging.eventgrid.implementation.models.CloudEvent");
+            });
+            comp.addImport("com.azure.core.models.CloudEvent");
+        });
+
+    }
+
+    private static final Map<String, String> replacementNames = new HashMap<String,String>() {
```

Comment:
nit: `static final`s should use upper case names - `REPLACEMENT_NAMES`.

Reply:
I'll fix this when I externalize it.